plugins/modules/locale_gen.py

```diff
@@ -25,9 +25,10 @@
         support: none
 options:
     name:
-        type: str
+        type: list
+        elements: str
         description:
-            - Name and encoding of the locale, such as "en_GB.UTF-8".
+            - Name and encoding of the locales, such as "en_GB.UTF-8".
```

Comment:
you should add note 'before version x.y.z this was a string'

Reply:
Added